Userland/Shell/PosixLexer.h

```diff
@@ -0,0 +1,489 @@
+/*
+ * Copyright (c) 2022, Ali Mohammad Pur <mpfard@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/DeprecatedString.h>
+#include <AK/GenericLexer.h>
+#include <AK/Variant.h>
+#include <AK/Vector.h>
+#include <Shell/AST.h>
+
+namespace Shell::Posix {
```

Comment:
newline here

Reply:
do we place newlines after namespace decls? my memory must be being silly again...
sure, will do.

Reply:
we do, clang-format doesn't, and I _really_ need to file an issue for that